### PR TITLE
Todo add

### PR DESCRIPTION
**Pull request by stanleySimeon**
# In this milestine I added: 
 
 - a function to add task
 - a function to edit a task
 - a function to remove a task

### CODE REVIEW

**Review by od-c0d3r — APPROVED** [review]
Asslam Alaikum @stanleySimeon ,😊

Hi👋🏽, 
Your project is complete!  
There is nothing else to say other than... it's time to merge it 😎 

# Congratulations! 🎉 
Happy coding!👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# Optional 
- [ ] Add `/dist` to your live link in your readme file to make it work 😉👍
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by od-c0d3r** [comment]
The review has been deleted, please wait for the next reviewer.

**Comment by stanleySimeon** [comment]
Thank you so much !

File: todo.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    this.saveToDo();
+  }
+
+  removeTodo = (index) => {
+    const updatedArray = this.LIST.filter((taskIndex) => taskIndex !== index);
+    updatedArray.forEach((item, index) => {
+      item.index = index + 1;
+    });
+    localStorage.setItem('todos', JSON.stringify(updatedArray));
+  };
```
Comment (FrederickMih) [inline todo.js:93]: - [ ] Again, in your case, a particular task can only be deleted when completed is set to true ie when the box is checked but I think we should be able to delete a task even if it's not completed. The `Clear all completed` button at the end of your todo content should be a better place to implement this.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @stanleySimeon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [OPTIONAL] In your README file, the link to the Live version of your project and GitHub link of the Author section is broken.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: todo.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+  getStoredTodos() {
+    this.LIST = JSON.parse(
+      localStorage.getItem('todos'),
+    );
+  }
+
+  editItem(text, index) {
+    this.LIST[index - 1].description = text.textContent;
+    this.saveToDo();
```
[the remark this answers is not in the document]
Comment (Whoistolu) [reply todo.js]: All good now 👍🏽

File: todo.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Whoistolu) [reply todo.js]: All good now 👍🏽

Comment (Whoistolu) [reply todo.js]: All good now 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @stanleySimeon,

Great job implementing the recommended changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**